mod/lockview.php

```diff
@@ -43,18 +47,20 @@ function lockview_content(App $a) {
 		killme();
 	}
 
-
-	if (($item['private'] == 1) && empty($item['allow_cid']) && empty($item['allow_gid'])
-		&& empty($item['deny_cid']) && empty($item['deny_gid'])) {
-
+	if ($item['private'] == 1
```

Comment:
The index "private" is used here, but won't be filled in photos, so this should generate a notice.